### PR TITLE
requested layout changes

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [Create Performance Evaluation Screen (Step 1) #5](https://github.com/nerubia/kh360-react/issues/5#issuecomment-1797147129)

### CODE REVIEW

**Review by Dev-Nino — APPROVED** [review]

File: src/features/admin/evaluation-administrations/create/CreateEvaluationForm.tsx [hunk 1]
```diff
@@ -92,7 +92,7 @@ export const CreateEvaluationForm = () => {
     <div className='flex flex-col gap-10'>
       <div className='flex flex-col md:w-1/2 gap-4'>
         <div>
-          <h1 className='text-lg font-bold'>Evaluation Name</h1>
+          <h1 className='font-medium'>Evaluation Name</h1>
```
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/create/CreateEvaluationForm.tsx:95]: use h2 🤔

File: src/features/admin/evaluation-administrations/create/CreateEvaluationForm.tsx [hunk 2]
```diff
@@ -102,7 +102,7 @@ export const CreateEvaluationForm = () => {
           />
         </div>
         <div className='flex flex-col'>
-          <h1 className='text-lg font-bold'>Evaluation Period</h1>
+          <h1 className='font-medium'>Evaluation Period</h1>
```
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/create/CreateEvaluationForm.tsx:105]: same

File: src/features/admin/evaluation-administrations/create/CreateEvaluationForm.tsx [hunk 3]
```diff
@@ -115,7 +115,7 @@ export const CreateEvaluationForm = () => {
                 max={formData.eval_period_end_date}
               />
             </div>
-            <h1 className='text-lg font-bold'>to</h1>
+            <h1 className='font-medium'>to</h1>
```
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/create/CreateEvaluationForm.tsx:118]: same

File: src/features/admin/evaluation-administrations/create/CreateEvaluationForm.tsx [hunk 4]
```diff
@@ -131,7 +131,7 @@ export const CreateEvaluationForm = () => {
           </div>
         </div>
         <div>
-          <h1 className='text-lg font-bold'>Evaluation Schedule</h1>
+          <h1 className='font-medium'>Evaluation Schedule</h1>
```
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/create/CreateEvaluationForm.tsx:134]: same

File: src/features/admin/evaluation-administrations/create/CreateEvaluationForm.tsx [hunk 5]
```diff
@@ -145,7 +145,7 @@ export const CreateEvaluationForm = () => {
                 error={validationErrors.eval_schedule_start_date}
               />
             </div>
-            <h1 className='text-lg font-bold text-center'>to</h1>
+            <h1 className='font-medium text-center'>to</h1>
```
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/create/CreateEvaluationForm.tsx:148]: same

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes ^^

**Review by jatnerubia — APPROVED** [review]
Cool

**Pull request merged**